docs/configuration.rst

```diff
@@ -425,6 +425,22 @@ Trace Context propagation
      version_added:
        v1.7.0: The ``b3multi`` propagation style was added and ``b3`` was deprecated in favor it.
 
+Metrics
+------
+
+.. ddtrace-configuration-options::
```

Comment:
We can move the runtime metrics configs to this block

Reply:
Done in ce2965f507bb7c929baeaeac57ef9650f6e2038e